mappings/net/minecraft/command/suggestion/SuggestionProviders.mapping

```diff
@@ -5,7 +5,9 @@ CLASS net/minecraft/class_2321 net/minecraft/command/suggestion/SuggestionProvid
 	FIELD field_10933 ASK_SERVER Lcom/mojang/brigadier/suggestion/SuggestionProvider;
 	FIELD field_10934 AVAILABLE_SOUNDS Lcom/mojang/brigadier/suggestion/SuggestionProvider;
 	FIELD field_10935 SUMMONABLE_ENTITIES Lcom/mojang/brigadier/suggestion/SuggestionProvider;
-	FIELD field_22245 ALL_BIOMES Lcom/mojang/brigadier/suggestion/SuggestionProvider;
+	FIELD field_22245 AVAILABLE_BIOMES Lcom/mojang/brigadier/suggestion/SuggestionProvider;
+	FIELD field_36376 AVAILABLE_FEATURES Lcom/mojang/brigadier/suggestion/SuggestionProvider;
+	FIELD field_36377 AVAILABLE_STRUCTURES Lcom/mojang/brigadier/suggestion/SuggestionProvider;
```

Comment:
Mind name the `context, builder` lambda param names for the two new providers' initialization code too?